.eslintrc

```diff
@@ -15,6 +16,7 @@
     "no-use-before-define": 2,
     "radix": 2,
     "semi": 2,
+    "block-spacing": 2,
```

Comment:
💄 I think the rules here are currently sorted alphabetically?

Also should this rule be in our styleguide?